documentation/book/ref_topic-operator-environment.adoc

```diff
@@ -0,0 +1,34 @@
+
+[id='topic-operator--environment-{context}']
+= Topic Operator environment
+
+When deployed standalone the Topic Operator can be configured via environment variables. 
```

Comment:
This will need rebase - it seems to be missing the TLS options.

Reply:
Per the IBM Styleguide:

via 
Use this term, which means “by way of,” to refer to routing data, such as in “Messages are routed via SMTP.” In other situations, try to replace “via” with one of the following synonyms: “across,” along,” “by,” “from,” “on,” or “through.”

So, we shouldn't be using it here I don't think.